templates/service-template.yml

```diff
@@ -832,6 +836,9 @@ objects:
             - name: service-account-limits-check-skip-org-id-list
               mountPath: /config/service-account-limits-check-skip-org-id-list.yaml
               subPath: service-account-limits-check-skip-org-id-list.yaml
+            - name: kas-fleet-manager-kafka-owner-list
+              mountPath: /config/kafka-owner-list.yaml
+              subPath: kafka-owner-list.yaml
```

Comment:
We also need to expose a parameter that takes a list of owners and use it to construct the ConfigMap
For example, see how it was done for the supported instance type:
 
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/250a94108e3741e55cc9b0f9511667d6cfdc8291/templates/service-template.yml#L160
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/250a94108e3741e55cc9b0f9511667d6cfdc8291/templates/service-template.yml#L549-L557
- https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/250a94108e3741e55cc9b0f9511667d6cfdc8291/templates/service-template.yml#L733-L735